guides/common/modules/ref_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,31 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {EL} 7 to {EL} 8 includes an upgrade from Ansible Engine 2.9 to Ansible Core 2.12.
+
+If you have custom Ansible content such as playbooks, job templates inside REX, roles and collections on disk, and you relied on modules being delivered by the Ansible RPM on {Project}, you have to take additional steps to adapt your Ansible installation or migrate your Ansible content.
+
+Ansible Core removed all non-essential modules shipped in the core installation.
+In terms of FQCN notation `namespace.collection.module`, you can continue to use `ansible.builtin.*`, but everything else is missing in Ansible Core.
+That means you will be no longer able to use non-builtin Ansible modules as you were used to and you have to get them from somewhere else, eventually.
+
+Current options to resolve this situation:
+
+* You download Ansible Galaxy from the AppStream RHEL 8 repository.
+If you need support regarding `/usr/bin/ansible-playbook`, you can reach out to Red Hat support, but we do not provide support for the content you try to execute with it.
+* You buy an Ansible subscription, configure `ansible-galaxy` to talk to `cloud.redhat.com` and use it to download supported collections from there.
+If you need support, you reach out to the Ansible team.
```

Comment:
Upstream users can get most of the same collections from `galaxy.ansible.com`, minus the enterprise support.
So can Satellite customers, they just need to be aware that that content is then not supported by anybody officially (the support-ability of the rest of the setup remains, just the content is unsupported).